keg_bouncer_test_app/model/entities.py

```diff
@@ -21,34 +21,35 @@ class UserMixin(flask_login.UserMixin):
     name = sa.Column(sa.Unicode(), nullable=False)
 
 
-class User(db.Model, UserMixin, mixins.PermissionMixin):
+class User(UserMixin, mixins.PermissionMixin, db.Model):
```

Comment:
If inheritance order matters, how about adding a comment regarding that to the class?